inc/admin.php

```diff
@@ -754,6 +762,23 @@ function js_templates() {
 		include plugin_dir_path( __FILE__ ) . '../tpl/js-templates.php';
 	}
 
+
+	/**
+	 * Need to render templates for new WP 4.8 widgets when not on the 'widgets' screen or in the customizer.
+	 */
+	function js_widget_templates() {
+		$screen = get_current_screen();
```

Comment:
Right now, this will be getting enqueued even on pages that don't have Page Builder active. 

Maybe `js_widget_templates` could be called inside `enqueue_admin_scripts`, after we've already checked that we're on the right page using `self::is_admin()` - https://github.com/siteorigin/siteorigin-panels/blob/develop/inc/admin.php#L227

This also means that when we have Page Builder in different places (like in Advanced Custom Fields, one day), it'd be enqueued properly.